src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -59,9 +60,11 @@ object WorkflowActor {
   }
 
   val TerminalStates = Vector(ExecutionStatus.Failed, ExecutionStatus.Done, ExecutionStatus.Aborted)
-  def isExecutionStateFinished(es: ExecutionStatus): Boolean = {
-    TerminalStates contains es
-  }
+
+  def isExecutionStateFinished(es: ExecutionStatus): Boolean = TerminalStates contains es
```

Comment:
I was going to say something else until I realized I was confused, but that raised another statement so here it is :)

I think this should be something like `isTerminal` to contrast with `isDone`. W/o knowing the internals I'd think finished and done were synonyms. It might also be good to swap done with something like succeeded just to make it even more clear.